pjmedia/src/pjmedia/sdp_neg.c

```diff
@@ -35,6 +54,11 @@ struct pjmedia_sdp_neg
     pj_bool_t             answer_with_multiple_codecs;
     pj_bool_t             has_remote_answer;
     pj_bool_t             answer_was_remote;
+#if PJMEDIA_SDP_NEG_MAINTAIN_SESSION_PT
+    unsigned              med_pt_map_cnt;
+    media_pt_map          med_pt_map[PJMEDIA_MAX_SDP_MEDIA];
```

Comment:
As the map is per media type, and currently we only "define" PTs for audio & video, perhaps `PJMEDIA_MAX_SDP_MEDIA` is not ideal for the array size as it does not really correspond to max media type number, note that currently the default is 16. IMO 2-3 is reasonable. To be safe, better put check somewhere that the map only applies to audio & video media.

Reply:
If we are using per stream mapping, then this should not be changed